frontend/src/index.scss

```diff
@@ -1,14 +1,372 @@
+:root {
+  --text-color: #333;
+  --background-color: #fff;
+  --terminal-background: #333;
+  --terminal-color: #fff;
+  --result-item-show-more-background: #fff;
+  --result-item-show-more-color: #666;
+  --search-sidebar-container-border-color: #ccc;
+  --search-sidebar-link-color: var(--text-color);
+  --search-sidebar-hovered-link-background: #eee;
+  --search-sidebar-selected-link-background: #0081c2;
+  --search-sidebar-selected-link-color: #fff;
+  /* Divider between search results */
+  --search-result-divider-line-color: #ccc;
+  /* Gets overwritten by bootstrap; has no effect */
+  --search-result-title-color: #08c;
+  /* Text color for the details in a search result (like name, version, outputs...) */
+  --search-result-short-details-color: #666;
+  /* When search results are expanded, they show tabbed help for installing the result. This is the border color for that tabbed thing.
+       It is only half of the border, the other half is styled by bootstrap. */
+  --search-result-expanded-tab-border: #ccc;
+  /* Color of the animation above the `Loading` text that is shown when searching */
+  --loader-color: #000;
+  /* Background color of the experimental badge next to the flakes search */
+  --info-label-background: #007dbb;
+  /* Text color for links. Probably overwritten by bootstrap */
+  --link-color: #007dbb;
+  /* Background color for the small badges with number in the sidebar */
+  --badge-background: #757575;
+}
+
+/* ------------------------------------------------------------------------- */
+/* -- Darkmode specific styles --------------------------------------------- */
+/* ------------------------------------------------------------------------- */
+
+@media (prefers-color-scheme: dark) {
+
+  // Define darkmode colors
+  :root {
+    --text-color: #ffffff;
```

Comment:
Usually you want white for important elements like headlines, important data, and interactive elements, and a less intense shade of grey for longer text. The general recommendation I found online is that the less intense color should not be grey, but the main text color with a bit of transparency (e.g. `rgba(255, 255, 255, 0.85)`) so it works on all backgrounds.

As the style sheet does not really have a separation between the different types of text, I chose to use white for all text but light grey is also a valid option. In #733 this will be changed to a `rgba(255, 255, 255, 0.85)`.